tests/fixtures/saas_fixtures.py

```diff
@@ -119,6 +119,27 @@ def connection_config_saas_without_saas_config(
     connection_config.delete(db)
 
 
+@pytest.fixture(scope="function")
+def connection_config_saas_with_custom_http_method(
+        db: Session,
+) -> Generator:
+    saas_config = example_saas_configs["mailchimp"].copy()
+    saas_config["endpoints"][0]["requests"]["read"]["method"] = "POST"
```

Comment:
Are we going to use this fixture elsewhere? If not I think we're better off making this line change in `test_generate_update_stmt_custom_http_method` below itself, that way we can set `"POST"` as a variable and assert both values are that variable? Otherwise can we use a constant var here? That way if the method ever changes we can change these two disconnected locations in one go.

Reply:
The `main` branch now has the Mailchimp config separate from a newly created `saas_example_config.yml`. I planned on this example config being the one we can add these new use cases to.

Reply:
I'll use `saas_example_config.yml` here too, after pulling latest main.